composer.bash

```diff
@@ -2,7 +2,7 @@
 # Icinga Web 2 Docker image | (c) 2020 Icinga GmbH | GPLv2+
 set -exo pipefail
 
-for d in icingaweb2 icinga-php/* icingaweb2/modules/*; do
+for d in icingaweb2 icingaweb2/modules/*; do
```

Comment:
Let's turn the question around: why do our nightlies require a Composer version our Docker image doesn't have?

Reply:
I'd say that doesn't matter if our Docker images don't even need to run Composer?